### ISSUE TITLE
Preset colors

### ISSUE THREAD

**Issue by Endermaster3690** [ScratchAddons/ScratchAddons#1807]
**Describe the new addon**

So what this does is that it adds a few preset colors, you can delete and edit preset colors.
You can also add preset colors.

It might really help!

**Comment by RedGuy12** [comment]
Colors for what?

**Comment by Endermaster3690** [comment]
> Colors for what?

Colors for the paint editor

**Comment by lisa-wolfgang** [comment]
This rings a bell of something suggested before, let me see if I can find it.

**Comment by Endermaster3690** [comment]
> This kind of reminds me of how the 2.0 paint editor had that color swatch:
> ![image](https://user-images.githubusercontent.com/43426138/110725956-221d4b00-81de-11eb-96a2-9744ad6d8bc7.png)
> It would be nice to have this with custom color slots, similar to the Google and Microsoft color pickers.

Indeed. If you want you can also make a pull request about this but you dont have to but it'd be nice if you did.

**Comment by Secret-chest** [comment]
The presets should be shown above the current color picker in the dropdown.

**Comment by Endermaster3690** [comment]
> The presets should be shown above the current color picker in the dropdown.

I was thinking of below, but that's good too.

**Comment by Endermaster3690** [comment]
With issue #1824 color picker at bottom, i think preset colors should be to the right of the picker.

**Comment by WorldLanguages** [comment]
Too many color picker suggestions at once, lol, we'll have a good time trying to make these addons compatible with each other

**Comment by Secret-chest** [comment]
Bump? Now we have table settings btw.

**Comment by Secret-chest** [comment]
Alternatively, we may support importing .gpl files.
